### ISSUE TITLE
Issue with AR standalone from Sep 9th Code and data branch w/o transportation tracker. 

### ISSUE THREAD

**Issue by TJTapajyoti** [NREL/celavi#37]
![image](https://user-images.githubusercontent.com/24482959/133485210-b6e72a3d-2339-4904-81f1-8cf4440464b6.png)

If its due to running old code version, please ignore.

**Comment by TJTapajyoti** [comment]
No other state run combination has this problem.

**Comment by rjhanes** [comment]
I ran commit 11f206f, filtering down to AR only, on the `national_data` branch and could not reproduce this error. I think it's due to an old routes file that doesn't have all the facilities in it.

I've added a couple commits to the `issue37` branch to adjust the way state filtering is done. With these new commits, if the data is being filtered down to the state level and the routes file is *not* being re-generated, only then does the (national) routes file need to be filtered. For these filtered runs, it's now feasible to re-generate the routes file for only the state(s) being run by setting `enable_data_filtering` and `run_routes` both to True. (Previously, `run_routes` = True meant that the entire national set of routes was always run.) I think defaulting to generating a new, single- or multi-state routes file for these smaller runs makes sense because it avoids this recurring issue, where the locations (etc) data has been updated but we are still using an outdated national routes file because it takes so long to compute.

So this fixes the immediate issue. I think we should still update the national routes file that's in OneDrive so we have the option to filter it down. I have some functionality around next use facilities that I'm still working on (PR #35) and once that's wrapped up I will re-generate the file and update the copy in OneDrive.
